src/schemas.py

```diff
@@ -45,7 +45,7 @@ class PeriodOfTime(BaseModel):
 class Column(BaseModel):
     name: str
     datatype: str
-    titles: str
+    # titles: str
```

Comment:
we'll need this, can see why you commented it out though, we're missing a context item for "titles" so its missing from the RDF, this should do it

```
    "titles": {
        "@id": "csvw:titles",
        "@language": "en"
    },
```

and just update your construct sparql to pull it in,